src/components/ResetPassword/resetpassword.jsx

```diff
@@ -12,8 +12,8 @@ export default function ResetPassword() {
     setEmail(e.target.value);
   };
   const [email, setEmail] = useState("");
+  // eslint-disable-next-line
   const re = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9] {1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-
   const PostData = () => {
     console.log("hi");
     if (re) {
```

Comment:
How this if statement checks the email?

Reply:
I have changed the pattern in which the email received will be validated. Thanks for bringing my notice to this